python/ray/serve/api.py

```diff
@@ -459,10 +459,15 @@ def _run(
     validate_route_prefix(route_prefix)
 
     if _local_testing_mode:
+        if logging_config is None:
+            logging_config = LoggingConfig()
+        elif isinstance(logging_config, dict):
+            logging_config = LoggingConfig(**logging_config)
```

Comment:
```suggestion
        if logging_config is None:
            logging_config = LoggingConfig(**(logging_config or {}))
```
maybe too much hackery :)